torchgeo/trainers/classification.py

```diff
@@ -338,12 +342,13 @@ def validation_step(
         """
         x = batch["image"]
         y = batch["label"]
+        batch_size = x.shape[0]
```

Comment:
do we have to store batch size as a variable, or can it just be directly in the `self.log` call?

Reply:
It can be directly used, but we're using it in multiple places and it's a little more clear to give it a name

Reply:
ah yeah true, I had only looked at the first BYOL task which only uses it once.